includes/components/class-attribute.php

```diff
@@ -307,6 +310,72 @@ public function register_models() {
 		}
 	}
 
+	/**
+	 * Validates unique model attributes.
+	 *
+	 * @param array  $errors Error messages.
+	 * @param object $model Model object.
+	 * @return array
+	 */
+	public function validate_unique_attributes( $errors, $model ) {
+		if ( empty( $errors ) ) {
```

Comment:
Please use guard clauses if the condition decides if the whole function should be continued, otherwise it adds extra spacing/nesting for the whole code